content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -61,18 +61,17 @@ Suppose a Kubernetes cluster is running a device plugin that advertises resource
 on certain nodes. Here is an example of a pod requesting this resource to run a demo workload:
 
 ```yaml
----
```

Comment:
These document separators [can] serve a purpose: they cue the reader into expecting that what follows is YAML.